packages/avatars/src/styled/StyledStatusIndicatorBase.ts

```diff
@@ -67,19 +67,24 @@ const sizeStyles = (props: IStyledStatusIndicatorProps) => {
   `;
 };
 
-const colorStyles = (props: IStyledStatusIndicatorProps) => {
-  let backgroundColor = getStatusColor(props.type, props.theme);
+const colorStyles = ({ theme, type }: IStyledStatusIndicatorProps) => {
+  const foregroundColor = getColor({
+    light: { hue: 'white' },
+    dark: { hue: 'neutralHue', shade: 1100 },
```

Comment:
Feels clunky, but it was in the design... waiting for their feedback to confirm better alternatives.